packages/webapp/src/hooks/useGenderOptions.jsx

```diff
@@ -12,13 +12,20 @@
  *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
  *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
  */
+import { useTranslation } from 'react-i18next';
+
+const genderOptions = [
+  { value: 'MALE', label: 'gender:MALE' },
```

Comment:
Just a small thing -- I thin i18n prefers namespacing as an option in `t()` rather than in keystring in t : https://www.i18next.com/principles/namespaces#sample

Reply:
Oh nice, for this situation that can be done very cleanly.